.mailmap

```diff
@@ -0,0 +1,46 @@
+Benjamin Ayliffe <benjamin.ayliffe@metoffice.gov.uk> benjamin.ayliffe <benjamin.ayliffe@metoffice.gov.uk>
+Benjamin Owen <benjamin.owen@bom.gov.au> benowen-bom <71359048+benowen-bom@users.noreply.github.com>
+Ben Fitzpatrick <ben.fitzpatrick@metoffice.gov.uk> Ben Fitzpatrick <ben@seven.powderham>
+Bruno P. Kinoshita <kinow@users.noreply.github.com> Bruno P. Kinoshita <kinow@users.noreply.github.com>
+Caroline Jones <caroline.jones@metoffice.gov.uk> caroline.jones <caroline.jones@metoffice.gov.uk>
+Caroline Jones <caroline.jones@metoffice.gov.uk> carolinejjones <caroline.jones@metoffice.gov.uk>
+Caroline Sandford <caroline.sandford@metoffice.gov.uk> Caroline Sandford <csand@localhost.localdomain>
+Caroline Sandford <caroline.sandford@metoffice.gov.uk> cgsandford <35029690+cgsandford@users.noreply.github.com>
+Carwyn Pelley <carwyn.pelley@metoffice.gov.uk> Carwyn Pelley <carwyn.pelley@metoffice.gov.uk>
+Chris Sampson <christopher.sampson@metoffice.gov.uk> BelligerG <44228125+BelligerG@users.noreply.github.com>
+Daniel Mentiplay <dmentipl@users.noreply.github.com> Daniel Mentiplay <dmentipl@users.noreply.github.com>
+Eleanor Smith <40183561+ellesmith88@users.noreply.github.com> ellesmith88 <40183561+ellesmith88@users.noreply.github.com>
+Fiona Rust <fiona.rust@metoffice.gov.uk> fiona.rust <fiona.rust@metoffice.gov.uk>
+Fiona Rust <fiona.rust@metoffice.gov.uk> fionaRust <fiona.rust@metoffice.gov.uk>
+Gavin Evans <gavin.evans@metoffice.gov.uk> gavinevans <gavin.evans@metoffice.gov.uk>
+James Canvin <41655739+nivnac@users.noreply.github.com> James Canvin <41655739+nivnac@users.noreply.github.com>
+Katharine Hurst <67630960+Kat-90@users.noreply.github.com> Kat <67630960+Kat-90@users.noreply.github.com>
+Kathryn Howard <kathryn.howard@metoffice.gov.uk> Kathryn.Howard <kathryn.howard@metoffice.gov.uk>
+Kathryn Howard <kathryn.howard@metoffice.gov.uk> Katie-Howard <kathryn.howard@metoffice.gov.uk>
+Laurence Beard <laurence.beard@metoffice.gov.uk> LaurenceBeard <laurence.beard@metoffice.gov.uk>
+Lucy Liu <lucy.liu@bom.gov.au> Lucy Liu <jliu176@gmail.com>
+Mark Baker <mark.baker@metoffice.gov.uk> Mark Baker <20327097+markysparks@users.noreply.github.com>
+Marcus Spelman <99179165+mspelman07@users.noreply.github.com> mspelman07 <99179165+mspelman07@users.noreply.github.com>
+Mark Worsfold <mark.worsfold@metoffice.gov.uk> MarkWorsfold <mark.worsfold@metoffice.gov.uk>
+Mark Worsfold <mark.worsfold@metoffice.gov.uk> MarkWorsfold <35810969+MarkWorsfold@users.noreply.github.com>
+Meabh NicGuidhir <meabh.nicguidhir@metoffice.gov.uk> meabh.nicguidhir <meabh.nicguidhir@metoffice.gov.uk>
+Meabh NicGuidhir <meabh.nicguidhir@metoffice.gov.uk> meabhnicguidhir <meabh.nicguidhir@metoffice.gov.uk>
+Neil Crosswaite <neil.crosswaite@metoffice.gov.uk> Neil <43375279+neilCrosswaite@users.noreply.github.com>
+Paul Abernethy <paul.abernethy@metoffice.gov.uk> Paul Abernethy <paul.abernethy@metoffice.gov.uk>
```

Comment:
My github userid is `PaulAbernethy`
```suggestion
Paul Abernethy <paul.abernethy@metoffice.gov.uk> PaulAbernethy <paul.abernethy@metoffice.gov.uk>
```

Reply:
but maybe it doesn't matter cause it uses my email address...

Reply:
How does the mailmap work? I assume the second reference is usually the github userid, but it seems that some people are listed using their github user id, e.g. BelligerG for Chris, LaurenceBeard for Laurence etc, but some people are listed using their normal id. Does the lack of consistency matter?

Reply:
It's a fair point. I autogenerated most of this, and I think it might depend on Git and GitHub setup. The name in the third column doesn't really do anything as far as I know... will look into it.

Reply:
Yep very little point to the third column 'commit name' - removed it for all and it simplifies the mailmap a lot, thanks! https://git-scm.com/docs/gitmailmap